packages/react-magma-dom/src/components/Stepper/Stepper.stories.tsx

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import { Button } from '../Button';
+import { Stepper, StepperProps, Step, StepperLayout } from './';
+import { Story, Meta } from '@storybook/react/types-6-0';
+import { Container } from '../Container';
+import { ButtonGroup } from '../ButtonGroup';
+import { InputType } from '../InputBase';
+import { Input } from '../Input';
+import { Flex, FlexAlignItems, FlexBehavior, FlexJustify } from '../Flex';
+import { LabelPosition } from '../Label';
+
+export default {
+  title: 'Stepper',
+  component: Stepper,
+  decorators: [
+    (Story, context) => (
+      <Container isInverse={context.args.isInverse} style={{ padding: '20px' }}>
+        <Story />
+      </Container>
+    ),
+  ],
+  argTypes: {
+    breakpoint: {
+      control: {
+        type: 'number',
+      },
+    },
+    breakpointLayout: {
+      control: {
+        type: 'select',
+        options: StepperLayout,
+      },
+    },
+    layout: {
+      control: {
+        type: 'select',
+        options: StepperLayout,
+      },
+    },
+    completionLabel: {
+      control: {
+        type: 'text',
+      },
+    },
+    stepLabel: {
+      control: {
+        type: 'text',
+      },
+    },
+    isInverse: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    testId: {
+      control: {
+        type: 'text',
+      },
+    },
+  },
+} as Meta;
+
+const Template: Story<StepperProps> = args => {
+  const [currentStep, setCurrentStep] = React.useState(0);
+
+  const [numberOfSteps, setNumberOfSteps] = React.useState('4');
+
+  console.log(`number of steps: ${numberOfSteps}`);
```

Comment:
Let's remove this console log now that it's working. This looks and works great by the way!